orchagent/vxlanorch.cpp

```diff
@@ -574,6 +574,7 @@ void VxlanTunnel::updateNextHop(IpAddress& ipAddr, MacAddress macAddress,
     if (it == nh_tunnels_.end())
     {
         nh_tunnels_[key] = {nh_id, 1};
+        nh_tunnels_[key].ref_count++;
```

Comment:
This section gets executed only if `key `added to `nh_tunnels` for the first time. So how will this get incremented in subsequent updates? May be i'm missing something.

Reply:
when call createNextHopTunnel will increase ref_count every times except create nh_tunnels first time.